src/main.rs

```diff
@@ -222,6 +223,16 @@ fn subject(category: Category, subject: String) -> Template {
     Template::render(page, &context)
 }
 
+#[get("/<dest>", rank = 9)]
+fn redirect(dest: redirect::Destination) -> Redirect {
+    Redirect::permanent(dest.uri)
+}
+
+#[get("/<_locale>/<dest>", rank = 9)]
+fn redirect_locale(_locale: redirect::Locale, dest: redirect::Destination) -> Redirect {
+    Redirect::permanent(dest.uri)
```

Comment:
I'd prefer to avoid permanent redirects, especially for the locale links. For example, if we add translations back into the website in the future it would be nice to redirect old links to the correct locale.

Reply:
Changed to use temporary redirects for links to non-en-US locales.